### DISCUSSION TITLE
cannot create a new branch

### DISCUSSION THREAD

**Discussion by KalpasWang** [Game-as-a-Service/Pandemic-Reign-of-Cthulhu#48] [Q&A] [answered]
@wingtkw @metalalive 
I created some UI demo page in my computer but I cannot create new branch in this repo to push, could you help me to see what's wrong here?

**Comment by wingtkw** [comment]
Sure let me see when I arrived home

**Comment by metalalive** [comment]
Hi , Kalpas, please describe the error message

**Comment by KalpasWang** [comment]
> Hi , Kalpas, please describe the error message you got

the problem is that i don't find create the "new branch" button in this repo's branches page.
I thought maybe I was not added as a collaborator for this repo

**Comment by metalalive** [comment, marked as answer]
> maybe I was not added as a collaborator for this repo

You are already a collaborator of the repo, with [read role](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization) defined by GitHub.

I've done giving you write role of the repo, please refresh the branch page to see whether there is a button `new branch`.

**Comment by KalpasWang** [comment]
@metalalive 
Thank you, I can create new branch now.🥹
